tracer/src/Datadog.Tracer.Native/rejit_preprocessor.cpp

```diff
@@ -456,8 +479,9 @@ ULONG RejitPreprocessor<RejitRequestDefinition>::PreprocessRejitRequests(
         {
             const auto target_method = GetTargetMethod(definition);
             const auto is_derived = GetIsDerived(definition);
+            const auto is_interface = GetIsInterface(definition);
 
-            if (is_derived)
+            if (is_derived || is_interface)
```

Comment:
Good we are re-using the derived code that already has the assembly ref optimization. So if a module doesn’t have an assembly ref to the module containing the interface we skips the whole module.